Config.psm1

```diff
@@ -76,6 +76,9 @@ function Get-AvailableConfigOptions {
                            The script names contain the information on when the script will be executed.
                            One can define only some of the hook scripts and it is not mandatory to define all of them.
                            If a script does not exist, it will not be executed."},
+        @{"Name" = "enable_administrator_account"; "GroupName" = "vm"; "DefaultValue" = $false; "AsBoolean" = $true
```

Comment:
change the GroupName to default (remove the GroupName key), as this option refers to the image, and not to the running vm.